### DISCUSSION TITLE
Docs on environment variables used in registry-core

### DISCUSSION THREAD

**Discussion by radhay-samagra** [Sunbird-RC/community#221] [General]
I tried finding any doc on configuring registry behaviors based on the environment variables introduces in the docker-compose file. I came across an attribute ```external_entity```. I was not able to find the exact use case for this. Can anyone point me to any doc where I can find these terminologies?

**Comment by tejash-jl** [comment]
@radhay-samagra I'm working on updating the documentation with the new env variables. 
FYI
external_entities: This field denotes external entities who are not part of sunbird-RC, but would need to access data from sunbird-rc. Those external entities need to be onboarded in keycloak as user/service accounts. After getting the token, they can access the data from sunbird-rc (only read access).

**Reply by radhay-samagra** [reply]
@tejash-jl 
Is there any doc for these properties maintained? 
Also can you help me with this particular attribute in AttestationPolicies 
```"attestorPlugin": "did:internal:ClaimPluginActor?entity=Teacher"```
Not sure I have seen this particular attribute anywhere mentioned.
